apps/bfd-pipeline/pom.xml

```
@@ -15,6 +15,11 @@
 		A parent POM for the bfd-pipeline-* projects, which migrate data from the CCW data warehouse to the Beneficiary FHIR Data systems.
 	</description>
 
+	<properties>
+		<awssdk.core.version>2.20.50</awssdk.core.version>
+		<awssdk.transfer.manager.version>2.20.50</awssdk.transfer.manager.version>
```

Personally, I'm still a little unclear on some of the finer points with managing poms. Could this instead inherit from the `bfd-parent`'s `pom.xml` and use `${aws-java-sdk-bom.version}`, e.g.

```suggestion
		<awssdk.core.version>${aws-java-sdk-bom.version}</awssdk.core.version>
		<awssdk.transfer.manager.version>${aws-java-sdk-bom.version}</awssdk.transfer.manager.version>
```

Or... generally inherit from `${aws-java-sdk-bom.version}` skipping `awssdk.core.version` and `awssdk.transfer.manager.version` all together?

---

I am as well, lol. I did consider that, but wasn't sure if it was always the case that a new release and bumping the `bom` version would also bump the other dependencies. I figured being explicit was probably better, but if it's the case that they'll always share the same version number, totally down to change it.

---

But. to answer your question - yes, I believe it can all be inherited.

---

I'm just going to update it...it's not like we wouldn't discover any errors at the time of building if the dependencies didn't exist.

---

[This](https://github.com/aws/aws-sdk-java-v2#importing-the-bom) it seem like you could go ahead and import the bom at the top level pom and just specify the services we want in the child modules **without** the identifying the version, and it will take care of itself. Is that your read?

---

...that's very obvious in hindsight. updated.